packages/core/src/logger/loggerConfiguration.ts

```diff
@@ -1,15 +1,19 @@
-export interface LoggingConfiguration {
-  isLoggingRequestInfo: boolean;
-  isLoggingRequestHeaders: boolean;
-  isLoggingRequestBody: boolean;
-  isLoggingResponseInfo: boolean;
-  isLoggingResponseHeaders: boolean;
-  isLoggingResponseBody: boolean;
-  headerFilters: string[];
-  headerLoggingPolicy: HeaderLoggingPolicy;
+import { Level, LoggerInterface } from '@apimatic/core-interfaces';
+
+export interface LoggingOptions {
+  logger?: LoggerInterface;
+  logLevel?: Level;
+  logRequest?: LogRequestOptions;
+  logResponse?: LogBaseOptions;
+}
+
+export interface LogBaseOptions {
```

Comment:
Rename options interface to match the existing pattern we follow in the SDK:

```suggestion
export interface HttpMessageLoggingOptions {
```

Reply:
The interfaces names were updated